engine/access/rpc/backend/backend.go

```diff
@@ -53,12 +57,14 @@ const DefaultConnectionPoolSize = 250
 //
 // It is composed of several sub-backends that implement part of the Access API.
 //
-// Script related calls are handled by backendScripts.
-// Transaction related calls are handled by backendTransactions.
-// Block Header related calls are handled by backendBlockHeaders.
-// Block details related calls are handled by backendBlockDetails.
-// Event related calls are handled by backendEvents.
-// Account related calls are handled by backendAccounts.
+// Script related calls are handled by Scripts.
+// Transaction related calls are handled by Transactions.
+// Block related calls are handled by BlockBase.
+// Block subscription calls are handled by SubscribeBlocks.
+// Event related calls are handled by Events.
+// Account related calls are handled by Accounts.
+// Execution result calls are handled by ExecutionResults.
+// Network and protocol snapshot calls are handled by Network.
```

Comment:
since we've split out pretty much everything, I don't think we need to enumerate them all here
```suggestion
```